### ISSUE TITLE
Error ignoring  dupe of package_name

### ISSUE THREAD

**Issue by deaw0003** [dtmateojr/dtmrepo#17]
After i run command
dtmrepo -augfv -d rhel6s-86_64
I found error about ignoring  dupe of package_name.That make these pakage not sync.

**Comment by dtmateojr** [comment]
That is not an error. That is a time-saver otherwise it would redownload hundreds of gigabytes of already downloaded packages every time.

**Comment by deaw0003** [comment]
But I fond if this meesage occur. Amount of total pakage after repolist dtmrepo itself isn't equl to repolist from CDN Red hat.
although I sync dtmrepo to latest (Red Hat 6)

**Comment by dtmateojr** [comment]
Use the mirror option

**Comment by deaw0003** [comment]
Where can i add mirror option ?

**Comment by dtmateojr** [comment]
I suggest that you completely read instructions.txt before asking another question.

**Comment by deaw0003** [comment]
Yes, i read instruction already.But i don't know where add to INSTALLDIR/etc/dtmrepo.conf.d that contain dtmrepo config file such as 
[control]
    releasever = 5Server
    arch = x86_64
    keep = 2
    protectrepo = os,optional

    [repos]
    os = rhel-5-server-rpms
    optional = rhel-5-server-optional-rpms
    supplementary = rhel-5-server-supplementary-rpms
    epel = epel-5-x86_64
    foo = local:/var/dtmrepo/foo/el5/x86_64
    misc = swalk:some-spacewalk-channel
    dev = mirror:dev-el5

or add mirror in /etc/yum.repo.d/ in server that need to repolist/upgrade such as
[epel-5-x86_64]
    name=epel-5
    mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-5&arch=x86_64
    failovermethod=priority
    gpgcheck=1
    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-5
